MojioSDK/Client/AuthClient.swift

```diff
@@ -306,6 +306,10 @@ public class AuthClient: NSObject, AuthControllerDelegate {
                 failure(response: response.result.value as? NSDictionary)
             }
         }
```

Comment:
You have to be in debug mode and add -DDEBUG to the flags for the print to show up.